mcs/class/Mono.Cairo/Mono.Cairo/Context.cs

```diff
@@ -49,7 +49,7 @@ public class Graphics : Context {
 
 	public class Context : IDisposable
 	{
-		IntPtr handle = IntPtr.Zero;
+		IntPtr handle;
```

Comment:
This is a no-op since objects are nulled on creation.  I will add this to the PR for clarity though.